public/css/category.css

```diff
@@ -0,0 +1,69 @@
+@import url('https://framer-ui.netlify.app/components/framer.css');
+
+:root{
+    --navbar-bg:#2C2929;
+    --img-bg:#2F3136;
+    --img-text:#f1f2f3;
+    --img-banner:#4F46E5;
+}
+
+.navbar-bg{
+    background: var(--navbar-bg);
+}
+
+.nav-zindex{
+    z-index: 9999;
+}
+.img-category{
+    min-width: 200px;
+    width: 19.5rem;
+    height: 100%;
+    border-radius: 10px;
+    background: var(--img-bg);
+    cursor: pointer;
+    min-height: 15.625rem;
+}
+
+.img-text-left{
+    position: absolute;
+    bottom: 1.2rem;
+    color: var(--img-text);
+    font-weight: 800;
+    padding-left: 10px;
+}
+.img-text-right{
+    position: absolute;
+    top: 1.2rem;
+    right: -0.2rem;
+    color: var(--img-text);
+    font-weight: 500;
+    font-size: 0.75rem;
+    padding: 5px;
+    background: var(--img-banner);
+    border-top-left-radius: 2px;
+    border-bottom-left-radius: 10px;
+
+}
+
+.qg-gap-5{
+    gap: 5rem;
+}
```

Comment:
remove this class as you have not used it in your Html page. The remaining code looks good to me. Can be merged.

Reply:
@yaswanthmyneni  Thanks. made the changes as u suggested.